index.bs

```diff
@@ -4093,11 +4093,12 @@ considered more trustworthy than the rest of the authenticator.
 Each authenticator stores a <dfn for=authenticator>credentials map</dfn>, a [=map=] from ([=rpId=], [=public key credential source/userHandle=]) to
 [=public key credential source=].
 
-Additionally, each authenticator has an AAGUID, which is a 128-bit identifier indicating the type (e.g. make and model) of the
-authenticator. The AAGUID MUST be chosen by the manufacturer to be identical across all substantially identical authenticators
-made by that manufacturer, and different (with high probability) from the AAGUIDs of all other types of authenticators.
-The AAGUID for a given type of authenticator SHOULD be randomly generated to ensure this. The [=[RP]=] MAY use the AAGUID to infer certain
-properties of the authenticator, such as certification level and strength of key protection, using information from other sources.
+Additionally, each authenticator has an Authenticator Attestation GUID or <dfn for=aaguid>AAGUID</dfn>, which is a 128-bit identifier indicating the type (e.g. make and model) of the
```

Comment:
nit: since the first two letters are already being spelt out, suggest also putting Globally Unique Identifier for completeness.

Reply:
fair enough